app/upgrades/nextupgrade/upgrades.go

```diff
@@ -238,6 +240,13 @@ func setInterchainTxsParams(ctx sdk.Context, paramsKeepers paramskeeper.Keeper,
 
 	bz := codec.MustMarshal(&currParams)
 	store.Set(interchaintxstypes.ParamsKey, bz)
+
+	wasmStore := ctx.KVStore(wasmStoreKey)
+	bzWasm := wasmStore.Get(wasmtypes.KeyLastCodeID)
+	if bzWasm == nil {
+		return fmt.Errorf("last code ID not found during the upgrade")
+	}
```

Comment:
There is no public method in wasmd keeper to use instead of this raw work with the storage?